Marlin/Configuration_adv.h

```diff
@@ -1284,11 +1284,12 @@
 // Additional options for DGUS / DWIN displays
 //
 #if HAS_DGUS_LCD
-  #define DGUS_SERIAL_PORT 2
+  #define DGUS_SERIAL_PORT 3
```

Comment:
DGUS might be attached to any board at all. We can't assume that '3' is any better than '2'.

Reply:
I am debuging FYSETC S6 , it use port 3. And it supports most for DGUS screen , at the moment , in the market. So i change it to 3.